### PR TITLE
update nancy CI build and switch to nightly

### PR DESCRIPTION
**Pull request by se3000**

### CODE REVIEW

File: .github/workflows/dependency-check.yml [hunk 1]
```diff
@@ -1,12 +1,23 @@
...
+      uses: actions/checkout@v2
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: ^1.15
+      id: go
+
+    - name: Write Go Modules list
+      run: go list -json -m all > go.list
```
Comment (topliceanu) [inline .github/workflows/dependency-check.yml:18]: Is this `go.list` something that nancy uses?!

File: .github/workflows/dependency-check.yml [hunk 2]
```diff
@@ -1,12 +1,23 @@
...
+
+    - name: Check vulnerabilities
+      uses: sonatype-nexus-community/nancy-github-action@b492e6567a301a914bd227d44bf10dcfe85438ad
       with:
-        target: go.sum
+        nancyCommand: sleuth -e bba60acb-c7b5-4621-af69-f4085a8301d0,d373dc3f-aa88-483b-b501-20fe5382cc80,4a444740-66e6-4268-9191-c12780af5d4e,5def94e5-b89c-4a94-b9c6-ae0e120784c2
```
Comment (topliceanu) [inline .github/workflows/dependency-check.yml:23]: What are these UUID for?

**Review by topliceanu — DISMISSED** [review]

File: .github/workflows/dependency-check.yml (hunk 1 above)
Comment (se3000) [reply .github/workflows/dependency-check.yml:18]: Yes, the normal command to run it locally with Go modules is `go list -json -m all | nancy sleuth`. Nancy also supports `dep` so there seem to be a couple of ways of coercing dependency managers to fit this standardized file, then the Nancy Github Action assumes that file is present.

File: .github/workflows/dependency-check.yml (hunk 2 above)
Comment (se3000) [reply .github/workflows/dependency-check.yml:23]: They are OSS Index IDs. I've added a comment on how to get more info as this list changes.

**Review by topliceanu — APPROVED** [review]

**Pull request merged**